src/site/content/en/react/add-manifest-react/index.md

```diff
@@ -84,11 +84,10 @@ file:
 
 To find out if all the properties are working correctly in this example:
 
-+  Mouse over the editor, press the **App** button, then press the **Show**
-   button to preview the app.
-+  Open the DevTools by pressing `CMD + OPTION + i` / `CTRL + SHIFT + i`.
-+  Click on the **Application** panel.
-+  Switch to the **Manifest** tab.
+{% Instruction 'preview' %}
+{% Instruction 'devtools' %}
+-  Click on the **Application** panel.
```

Comment:
Should this be a short code?

Reply:
ah yeah, `{% Instruction 'devtools-application' %}` I think.